client/integrationTest/nodeTests/integrationNode.test.ts

```diff
@@ -41,12 +41,8 @@ import { UserInputUtil } from '../../src/commands/UserInputUtil';
 import { CommandUtil } from '../../src/util/CommandUtil';
 import { PackageRegistryEntry } from '../../src/packages/PackageRegistryEntry';
 import { PackageRegistry } from '../../src/packages/PackageRegistry';
-import { PeerTreeItem } from '../../src/explorer/runtimeOps/PeerTreeItem';
-import { FabricGatewayRegistry } from '../../src/fabric/FabricGatewayRegistry';
-import { FabricGatewayRegistryEntry } from '../../src/fabric/FabricGatewayRegistryEntry';
 import { GatewayTreeItem } from '../../src/explorer/model/GatewayTreeItem';
 
-const should: Chai.Should = chai.should();
```

Comment:
Are you sure - if you don't call `chai.should()`, it won't add the `.should.something.something...` methods to every object.

Reply:
you only need that line if you need to do `should.not.exist(blah)`

Reply:
it still adds the `.should.something` otherwise my tests would fail ;)